test/OpenTelemetry.Instrumentation.AWS.Tests/TestRequest.cs

```diff
@@ -13,9 +13,13 @@
 
 namespace OpenTelemetry.Instrumentation.AWS.Tests;
 
-internal class TestRequest(ParameterCollection parameters) : IRequest
+internal class TestRequest : IRequest
 {
-    private readonly ParameterCollection parameters = parameters;
+    private readonly ParameterCollection parameters;
```

Comment:
@ppittle Could you add a newline after this? This is the [error](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1516.md): https://github.com/open-telemetry/opentelemetry-dotnet-contrib/actions/runs/7999444791/job/21847990924?pr=1572#step:4:239

Reply:
Looks like @Kielek fixed it for me :)